react-rendering-lists/my-react-rendering-list/src/PokemonList.jsx

```diff
@@ -0,0 +1,14 @@
+const pokedex = [
```

Comment:
The instructions tell you to declare this in App and pass it as a parameter.
Other than this, the code all looks correct.